src/probnum/quad/bq_methods/policies/_policy.py

```diff
@@ -0,0 +1,108 @@
+"""Abstract base class for BQ acquisition policies."""
+
+import abc
+
+import numpy as np
+
+from ..acquisitions import Acquisition
+from ..bq_state import BQState
+
+
+class Policy(abc.ABC):
+    """An abstract class for acquisition functions for Bayesian quadrature.
+
+    Parameters
+    ----------
+    batch_size :
+        Size of batch that is found in every iteration of calling the policy.
+    is_deterministic :
+        Is the policy a deterministic function of its arguments or stochastic (i.e.
+        sampling-based)?
+    """
+
+    def __init__(self, batch_size: int, is_deterministic: bool) -> None:
+        self.batch_size = batch_size
+        self._is_deterministic = is_deterministic
```

Comment:
I don't get what should go, and why. Please clarify.

Reply:
Since `Policy` is an abstract class anyways, you can do 

```
@property    
@abc.abstractmethod
def is_deterministic(self) -> None:
    pass
```

and not use the instance variable `self._is_deterministic`. Then, the child class simply implements it, since that value is fixes anyways.